plugins/event-watcher/pkg/producers/kyverno_event_handler.go

```diff
@@ -7,16 +7,33 @@ import (
 
 	"log/slog"
 
+	"github.com/allegro/bigcache/v3"
 	"github.com/fairwindsops/insights-plugins/plugins/event-watcher/pkg/client"
 	"github.com/fairwindsops/insights-plugins/plugins/event-watcher/pkg/models"
+	"github.com/fairwindsops/insights-plugins/plugins/event-watcher/pkg/utils"
 
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/fields"
 )
 
 const (
-	EventVersion = 1
+	EventVersion                        = 1
+	KyvernoPolicyViolationFieldSelector = "involvedObject.kind=ClusterPolicy,reason=PolicyViolation"
 )
 
+var alreadyProcessedKyvernoPolicyViolationIDs *bigcache.BigCache
+
+func init() {
+	var err error
+	config := bigcache.DefaultConfig(60 * time.Minute)
+	config.HardMaxCacheSize = 256 // 512MB
+	alreadyProcessedKyvernoPolicyViolationIDs, err = bigcache.New(context.Background(), config)
+	if err != nil {
+		slog.Error("Failed to create bigcache", "error", err)
```

Comment:
unless we can safely run without `bigcache`, we should halt the program if it fails here. Either by `panic` or removing the `init` function and handling normally